scripts/loadWebhooks.js

```diff
@@ -20,13 +19,14 @@ const main = async () => {
   })
 
   // Fetch old nodes
-  const { data: oldNodes } = await client.queryAll(Q(NODES_DOCTYPE))
+  const oldNodes = await client.queryAll(Q(NODES_DOCTYPE))
 
   // Delete them
   await client.collection(NODES_DOCTYPE).destroyAll(oldNodes)
 
   // Create a new doc for each instance
-  await client.collection(NODES_DOCTYPE).saveAll(newWebhooks)
+  // Using collection to force the doctype
```

Comment:
Elements don't have a `_type` attribute at this moment. Either I add this attribute manually or I use `collection`

Reply:
Are you sure? The `_type` should be set by DocumentCollection: https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/DocumentCollection.js#L39
If you don't have it, it might be a bug

Reply:
It's because in this case, the documents are created by the script, which fetches the webhooks and puts them in plain JS object. I could add the `_type` in the script

Reply:
Ah, my bad, I mistakenly thought `oldNodes` was updated, not `newWebhooks`. In this case, yes, I think you could manually add the `_type` :+1: